### PR TITLE
Update submit_new_license.html

### PR DESCRIPTION
**Pull request by jlovejoy**
Update headings as per SPDX-legal discussion on Aug 25th

Signed-off-by: Jilayne Lovejoy

### CODE REVIEW

**Review by goneall — APPROVED** [review]
@jlovejoy - Thanks for pulling this together - you're now an official DJango programmer ;)

Were you able to test these out on your local machine?  There are 2 ways to setup a local environment to run this on your local computer.  That way, you can check out it out visually before it is fully deployed.

**Comment by jlovejoy** [comment]
I have not and I have no idea as to how to do that!

Also, there are some more changes that I think we might want to make. Unless you think it's better to do stuff more incrementally?

**Comment by goneall** [comment]
> Also, there are some more changes that I think we might want to make. Unless you think it's better to do stuff more incrementally?

It would be nice to get all the changes into one branch or one PR and then test it out.  We could setup some time and I can test it on a zoom call and screen share it with you and anyone else who's interested.  I just want to make sure everything looks good before it is deployed.  It takes me about an hour to deploy, and only a few minutes to update the UI when it's running on my local machine.

Ideally, we could get you setup to test locally on your machine, but we could do that later.

**Comment by jlovejoy** [comment]
@goneall @rtgdk  - now that #389 is merged.. do I need to re-do this one since it deals with the same file??

**Comment by rtgdk** [comment]
@jlovejoy No need to redo if there are no changes on the same line number as #389. If changes are on different line no. then git will handle it automatically.

I would suggest to `pull/rebase` latest master into this branch and test that the page looks like just what you wanted. Overall, changes LGTM and we can merge on confirmation.

**Comment by goneall** [comment]
@jlovejoy - Let me know if you'd like any help setting up a local environment for testing.

File: src/app/templates/app/submit_new_license.html [hunk 1]
```diff
@@ -22,11 +22,11 @@
...
       <div class="form-group">
           <label for="branchName">Branch Name:</label>
-          <span class="glyphicon glyphicon-info-sign" data-toggle="tooltip" title="Pull Request will be made from this branch to master branch" style="font-size:12px;"></span>
+          <span class="glyphicon glyphicon-info-sign" data-toggle="tooltip" title="Pull Request will be made from this branch to main branch" style="font-size:12px;"></span>
           <input type="text" class="form-control" id="branchName" name="branchName">
       </div><br>
       <div class="checkbox">
-          <label><input type="checkbox" id="updateUpstream"> Update with upstream master </label> <span class="glyphicon glyphicon-info-sign" data-toggle="tooltip" title="Select this if you want to update your forked repo with the upstream repo, before making the pull request" style="font-size:12px;"></span>
+          <label><input type="checkbox" id="updateUpstream"> Update with upstream main </label> <span class="glyphicon glyphicon-info-sign" data-toggle="tooltip" title="Select this if you want to update your forked repo with the upstream repo, before making the pull request" style="font-size:12px;"></span>
```
Comment (jlovejoy) [inline src/app/templates/app/submit_new_license.html:29]: do we want the hover over items in this window?

File: src/app/templates/app/submit_new_license.html [hunk 2]
```diff
@@ -151,7 +135,7 @@
```
Comment (jlovejoy) [inline src/app/templates/app/submit_new_license.html:135]: Check License Diff is commented out - why? (did this ever work, I can't remember?)

**Comment by jlovejoy** [comment]
> @jlovejoy - Let me know if you'd like any help setting up a local environment for testing.

I think I'm ready! and have a couple other questions/comments for you to discuss!

**Comment by jlovejoy** [comment]
note as per Oct 27th legal call:
- also remove hover-overs in Github PR dialog box for XML editor
- question on branch field - what is supposed to go here? (can we simplify)

File: src/app/templates/app/submit_new_license.html (hunk 2 above)
Comment (rtgdk) [reply src/app/templates/app/submit_new_license.html:135]: Yes, this was disabled in April this year following this issue - https://github.com/spdx/spdx-online-tools/issues/355 

We can take a call to re-enable this on this page after this gets in https://github.com/spdx/spdx-online-tools/pull/392 which improves matching

File: src/app/templates/app/submit_new_license.html (hunk 1 above)
Comment (rtgdk) [reply src/app/templates/app/submit_new_license.html:29]: I think we can remove hover from here as well and keep all help text in the same text

**Review by rtgdk — APPROVED** [review]
LGTM

**Comment by goneall** [comment]
@jlovejoy I merged in the recent changes to the master branch which should fix the problems you were running into with docker - just pull the latest and it should work

File: src/app/templates/app/submit_new_license.html (hunk 1 above)
Comment (jlovejoy) [reply src/app/templates/app/submit_new_license.html]: I removed these in last commit

**Comment by jlovejoy** [comment]
@goneall @rtgdk - in the Github PR dialog box - do either of you know why there is a "commit message" field and a "PR body text" field? Don't these end up being the same thing?

**Comment by jlovejoy** [comment]
Okay, figured out how to test this and made some updates to the text describing the fields. I'm still debating the short identifier field which currently says:
"Suggest short identifier (consisting of: ASCII letters (A-Z, a-z), digits (0-9), full stops (.) and hyphens (-). No spaces."

Which is kind of long and clunky. What if instead it simply said:

"Suggest short identifier according to the guidance in section B (link here: https://github.com/spdx/license-list-XML/blob/main/DOCS/license-fields.md )

@swinslow - thoughts?

**Comment by jlovejoy** [comment]
I dont think I can test the Github PR dialog though.

**Comment by goneall** [comment]
> in the Github PR dialog box - do either of you know why there is a "commit message" field and a "PR body text" field? Don't these end up being the same thing?

@jlovejoy There are actually different fields in Github.  Most of the time they are the same, but sometimes people may want them to differ.  There is a different standard for what the content and length of these fields should be - but for our purposes, they will most likely be the same.

I have no problem replacing both with a "comment and PR" field, but I also don't think it is too bad the way it is since they are different fields and some people may complain if we combine them.

One cool way to handle it would be to automatically populate the send field with the content of the first - which I think can be done in JavaScript - but I won't have time to code this up - very busy this week.

**Comment by goneall** [comment]
> I dont think I can test the Github PR dialog though.

When you're all done, I can test this part out before pushing to production (I just have to remember to do it!).

**Comment by swinslow** [comment]
> Okay, figured out how to test this and made some updates to the text describing the fields. I'm still debating the short identifier field which currently says:
> "Suggest short identifier (consisting of: ASCII letters (A-Z, a-z), digits (0-9), full stops (.) and hyphens (-). No spaces."
> 
> Which is kind of long and clunky. What if instead it simply said:
> 
> "Suggest short identifier according to the guidance in section B (link here: https://github.com/spdx/license-list-XML/blob/main/DOCS/license-fields.md )

@jlovejoy I'm kind of indifferent here, to be honest? The link to section B is more complete, though it also requires the requester to go there and read the guidance. I'm fine with either since ultimately this is just a suggestion by the submitter, and the Legal Team will review / revise as we feel appropriate in any case.

**Comment by jlovejoy** [comment]
ok, @goneall - I think this is ready to go from my end!

**Comment by goneall** [comment]
@jlovejoy @rtgdk I just tested this out on my local machine and I'm not able to submit.

The reason is the call to `form.is_valid()` is returning false.

**Comment by goneall** [comment]
Found the issue - we removed email from the JavaScript, but it is still required on the back-end.

I just pushed a commit which fixes this

**Comment by goneall** [comment]
I noticed another issue when reviewing the submitted issue.  It includes the field Standard License Header even though it was removed from the UI.

@jlovejoy Is this OK?  If so, I'll go ahead and merge.

```
1. License Name: Test license - please ignore
2. Short identifier: test-license
3. License Author or steward: Unknown
4. Comments: Test license - please ignore
5. Standard License Header:
6. License Request Url: http://localhost:8000/app/license_requests/4
7. URL(s): http://example.com/
8. OSI Status: Not Submitted
9. Example Projects: http://example1.com/

```

**Comment by jlovejoy** [comment]
Thanks @goneall and sorry for the extra work.

As for the StandardHeader - I think all references to that can be removed, if possible - sounds like I missed something, if I understand correctly?

**Comment by goneall** [comment]
> As for the StandardHeader - I think all references to that can be removed, if possible - sounds like I missed something, if I understand correctly?

@jlovejoy yes - it the email sending code, we just need to remove the line and probably change the numbering.  It's probably in the file views.py.

**Comment by goneall** [comment]
@jlovejoy  - since I added a commit, be sure to do a git pull before making any changes

**Pull request merged**